### PR TITLE
🏳️  Nft carousel

### PR DESCRIPTION
**Pull request by WRadoslaw**
#3775 

⚠️ There are still few issues:

- Idk why, but on first page load items on edges (when at item 0 -> item on the left) are displaying as skeletons even tho they are loaded. To showcase:
1. Go into playground
2. Choose carousel and hit F5
3. Navigate left (item should be a skeleton and on entry will load normally)
4. After changing to any other playground and coming back to carousel (without F5) same flow as before will work perfectly

- Sometimes there is a gap between fade and video in the background even tho both elements are the same width

### CODE REVIEW

File: packages/atlas/src/api/hooks/nfts.ts [hunk 1]
```diff
@@ -174,3 +175,12 @@ export const useNftHistory = (
     ...rest,
   }
 }
+
+export const useFeaturedNfts = () => {
```
Comment (WRadoslaw) [inline packages/atlas/src/api/hooks/nfts.ts:199]: This is only a placeholder till orion work is merged

File: packages/atlas/src/components/Gallery/Gallery.tsx [hunk 2]
```diff
@@ -12,14 +13,13 @@ export type GalleryProps = PropsWithChildren<{
...
   className?: string
   seeAllUrl?: string
+  children: ReactNode[] | ReactNode
 }> &
-  CarouselProps
+  Omit<CarouselProps, 'children'>
 
-export const Gallery: FC<GalleryProps> = ({ title, className, seeAllUrl, ...carouselProps }) => {
+export const Gallery: FC<GalleryProps> = ({ title, className, seeAllUrl, children, ...carouselProps }) => {
   // TODO: this is the only place in the app that requires refs to buttons. Once we refactor this component, we can remove forwardRef from buttons
```
Comment (WRadoslaw) [inline packages/atlas/src/components/Gallery/Gallery.tsx]: Remove

File: packages/atlas/src/components/Gallery/Gallery.tsx [hunk 3]
```diff
@@ -42,29 +42,29 @@ export const Gallery: FC<GalleryProps> = ({ title, className, seeAllUrl, ...caro
...
       </GridHeadingContainer>
-      <Carousel
-        {...carouselProps}
-        prevArrowRef={prevArrowRef}
-        nextArrowRef={nextArrowRef}
-        ref={carouselRef}
-        itemWidth={350}
-      />
+      <Carousel {...carouselProps}>
+        {/* eslint-disable-next-line @typescript-eslint/ban-ts-comment */}
```
Comment (WRadoslaw) [inline packages/atlas/src/components/Gallery/Gallery.tsx]: Fix

File: packages/atlas/src/components/NftCarousel/components/MarketplaceCarouselCard/NftCarouselDetails.tsx [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+                )}
+                {nftDetails.topBid && (
+                  <DetailsContent
+                    tileSize="big"
+                    caption="TOP BID"
+                    content={nftDetails.topBid}
+                    icon={<JoyTokenIcon size={16} variant="regular" />}
+                  />
+                )}
+                <div>{nftDetails.endsAt?.toLocaleString()}</div>
```
Comment (WRadoslaw) [inline packages/atlas/src/components/NftCarousel/components/MarketplaceCarouselCard/NftCarouselDetails.tsx]: Implement countdown

File: packages/atlas/src/components/Carousel/Carousel.tsx [hunk 5]
```diff
@@ -1,110 +1,34 @@
...
-import { GliderProps, useGlider } from '@/components/Glider'
-
-import { Container, Dots, GliderContainer, Track } from './Carousel.styles'
+import { StyledSwiper } from '@/components/Carousel/Carousel.styles'
 
+export type SwiperInstance = SwiperType
 export type CarouselProps = {
   className?: string
-  arrowPosition?: number
+  // arrowPosition?: number
```
Comment (attemka) [inline packages/atlas/src/components/Carousel/Carousel.tsx]: lets delete it completely if we'll not going to use it?

**Review by attemka — CHANGES_REQUESTED** [review]
works well, I've just noticed 2 things:
1. keyboard arrow controls doesn't work anymore. I don't think that many users will use that, but lets enable it if its a quick fix.
2. we need to add the callback in the end of the video playback to switch to the next video. Currently video just ends

**Review by attemka — APPROVED** [review]

**Pull request merged**